test/controllers/measurements_controller_test.rb

```diff
@@ -113,6 +113,30 @@ class FilterMeasurementsControllerTest < ActionDispatch::IntegrationTest
       get measurements_url, params: { created_after: 1.day.ago }, env: public_auth_header
       assert_equal(0, measurements.count)
     end
+
+    test 'should filter in UTC if given CET' do
+      m1 = create(:measurement, created_at: '2018-01-29T20:02:00+01:00') # 20:02 in Rapperswil
+
+      get measurements_url, params: { created_after: '2018-01-29T20:00:00+01:00',
+                                      created_before: '2018-01-29T20:04:00+01:00' },
+                            env: public_auth_header
+
+      assert_equal(1, measurements.count)
+      assert_equal(m1.id, measurements.first['id'])
+      assert_equal(m1.created_at, Time.zone.iso8601(measurements.first['created_at']))
+    end
+
+    test 'should not stumble over garbage numbers in the parameters' do
+      create_list(:measurement, 4)
+
+      get measurements_url, params: { last: 'garbage' }, env: public_auth_header
+      assert_response :success
```

Comment:
Hm, don't you think a "HTTP 400 Bad Request" would be better than simply ignoring the value? Might save some debugging time for users of the API (fail early).

Reply:
Ok. But then let's do it properly validate all filter parameters.

Reply:
I consider this to belong to this issue: https://github.com/coredump-ch/water-sensor-api/issues/38